### ISSUE TITLE
Improve Test Harness documentation

### ISSUE THREAD

**Issue by billwambua** [medic/cht-conf-test-harness#157]
The documentation for the Test Harness is too terse and is hard to follow at times. It also needs a larger variety of examples especially for people just getting into using it.

Right now the best and fastest way to learn how to use it is to check people who have used it before.

The documentation for the Test Harness is generated from the incode comments using Doxygen I believe.

Solution for this is to add more elaborate in code documentation.

**Comment by kennsippell** [comment]
Agreed.  Check out https://github.com/medic/app-services-team/issues/32 and the resulting [Capacity Building Guide](https://docs.google.com/presentation/d/1v1O5d_DElXTh961I4ciurFD40rGu-yV2grovP_1Kd_c/edit#slide=id.g109cf2b0043_1_382) written by @binokaryg . Tons of good content there and I believe he is working on getting that into standard public formats and integrated with https://docs.communityhealthtoolkit.org/
